### ISSUE TITLE
java.lang.RuntimeException: Failed second attemp to check library integrity

### ISSUE THREAD

**Issue by Audition-CSBlock** [OopsieWoopsie/Spicord#12]
I compiled the source code from commit 6cf2af2aa4268a960b338be0e08028be4efb8a39 (most recent version release and thus in my opinion, a version that'll work and isn't in beta)

I get this error when running my compiled jar on my paper-184 server:
```java
java.lang.RuntimeException: Failed second attemp to check library integrity
        at eu.mcdb.spicord.LibraryLoader.sha1Check(LibraryLoader.java:112) ~[?:?]
        at eu.mcdb.spicord.LibraryLoader.sha1Check(LibraryLoader.java:113) ~[?:?]
        at eu.mcdb.spicord.LibraryLoader.downloadLibraries(LibraryLoader.java:68) ~[?:?]
        at eu.mcdb.spicord.SpicordLoader.load0(SpicordLoader.java:54) ~[?:?]
        at eu.mcdb.spicord.SpicordLoader.<init>(SpicordLoader.java:47) ~[?:?]
        at org.spicord.bukkit.SpicordBukkit.lambda$onEnable$0(SpicordBukkit.java:41) ~[?:?]
        at org.spicord.bukkit.SpicordBukkit.onEnable(SpicordBukkit.java:43) ~[?:?]
        at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:263) ~[patched_1.16.2.jar:git-Paper-184]
        at org.bukkit.plugin.java.JavaPluginLoader.enablePlugin(JavaPluginLoader.java:380) ~[patched_1.16.2.jar:git-Paper-184]
        at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:483) ~[patched_1.16.2.jar:git-Paper-184]
        at org.bukkit.craftbukkit.v1_16_R2.CraftServer.enablePlugin(CraftServer.java:499) ~[patched_1.16.2.jar:git-Paper-184]
        at org.bukkit.craftbukkit.v1_16_R2.CraftServer.enablePlugins(CraftServer.java:413) ~[patched_1.16.2.jar:git-Paper-184]
        at net.minecraft.server.v1_16_R2.MinecraftServer.loadWorld(MinecraftServer.java:468) ~[patched_1.16.2.jar:git-Paper-184]
        at net.minecraft.server.v1_16_R2.DedicatedServer.init(DedicatedServer.java:241) ~[patched_1.16.2.jar:git-Paper-184]
        at net.minecraft.server.v1_16_R2.MinecraftServer.w(MinecraftServer.java:939) ~[patched_1.16.2.jar:git-Paper-184]
        at net.minecraft.server.v1_16_R2.MinecraftServer.lambda$a$0(MinecraftServer.java:177) ~[patched_1.16.2.jar:git-Paper-184]
        at java.lang.Thread.run(Thread.java:834) [?:?]```

Any help would be appreciated :)

**Comment by OopsieWoopsie** [comment]
That commit broke the class loader, I fixed that in the next commit 99719a5

**Comment by Audition-CSBlock** [comment]
@OopsieWoopsie username checks out haha, don't worry, it's all good :)
